File: C/10 - numberArmstrongOrNot.c
#include <stdio.h>
#include <math.h>
int intpow(int a, int b){
    int i = 0, r = 1;
    while (i < b){
        r *= a;
        i++;
    }
    return r;
}
int main() {
    int original, buffer,reminder, sum = 0;
    printf("Enter any number: ");
    scanf("%d", &original);
    buffer = original;
    while (buffer > 0) {
        reminder = buffer % 10;
        sum += intpow(reminder, 3);
        buffer /= 10;
    }
    if (original == sum) { printf("%d is an Armstrong number", original); }
    else { printf("%d is not an Armstrong number", original); }
    return 0;
}

File: C#/10 - numberArmstrongOrNot.cs
using System;

public class numberArmsstrongOrNot
{
    public static void Main(string[] args)
    {
        int original, buffer, reminder, sum = 0;
        Console.Write("Enter any number: ");
        original = Convert.ToInt32(Console.ReadLine());
        buffer = original;
        while (buffer > 0)
        {
            reminder = buffer % 10;
            sum += (int)Math.Pow(reminder, 3);
            buffer /= 10;
        }
        if (original == sum)
        {
            Console.WriteLine($"{original} is an Armstrong number");
        }
        else
        {
            Console.WriteLine($"{original} is not an Armstrong number");
        }
    }
}